### PR TITLE
re-arrange content on index page a little bit

### PR DESCRIPTION
**Pull request by rossPatton**
just adds a 'looking for a tech union?' blurb on the home page where the BLM blurb currently is. moved BLM message a little bit down, moved the security tips link into the nav

tech union blurb just links to the list corey put together. happy to link to something else if we have anything more extensive, but i feel like it's a good synopsis of the state of things

i'm sure we'll want translations, but i figure they can be done separately. or people can just push to this branch

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ permalink: /
 <h2 class="marg-b-4">{% t home.summary %}</h2>
 
 <div class="marg-b-3 p-3 secondaryBg">
-  <h3>{% t home.black_lives_matter.title %}</h3>
-  <p>{% t home.black_lives_matter.description %}</p>
+  <h3>{% t home.looking_for_union.title %}</h3>
```
Comment (shushugah) [inline index.md:11]: Given this is not translated, and it's English language centric to begin with I'd keep it as. And this minimizes the changes to other languages, while keeping this change English only, as that's our goal. Consistency is not as important, as having localized language knowledge available 

{% if site.lang == 'en' %}
  <h3>{% t home.looking_for_union.title %}</h3>
<p>{% t home.looking_for_union.description %}</p>
{%else %}
  <h3>{% t home.black_lives_matter.title %}</h3>
  <p>{% t home.black_lives_matter.description %}</p>
{% endif %}

File: _i18n/en.yml [hunk 2]
```diff
@@ -13,9 +13,9 @@ home:
   title: |
     <span class="red">Worker</span> power in the tech industry
   summary: Guided by our vision for an inclusive & equitable tech industry, TWC organizes to build worker power through rank & file self-organization and education.
-  black_lives_matter:
-    title: Black Lives Matter
-    description: We encourage Tech Workers to stand with the Black Lives Matter movement as they fight for justice. To that end, we've put together <a href="/security">a list of security guides here</a> you can use to help protect yourself and your comrades.
+  looking_for_union:
+    title: Looking for a Tech Worker Union?
+    description: While TWC is strongly affiliated with several unions around the world, and counts many union members and organizers in our ranks, we are not a union ourselves. But if you are interested in unionizing, we can help! If you're curious what's out there, check out <a href="https://organize.fyi/">this (rough) list</a> of active and ongoing labor efforts in both tech and adjacent industries.
```
Comment (shushugah) [inline _i18n/en.yml:18]: I'd remove the word `But`: 

`If you are interested in unionizing...`

File: _i18n/en.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ home:
     description: We organize for activism, civic engagement and education in the Bay Area, Seattle, across the United States and around the world.
   who_we_support:
     title: Who we support
-    description: We work in solidarity with existing movements towards social justice, workers' rights, and economic inclusion.
+    description: We work in solidarity with existing movements towards social justice, workers' rights, and economic inclusion. Black Lives Matter!
```
Comment (shushugah) [inline _i18n/en.yml:27]: I'd remove Black Lives Matter here. It's weirdly out of context, and looks tokenizing/cheap. Thoughts?

**Review by shushugah — CHANGES_REQUESTED** [review]

File: _i18n/en.yml (hunk 3 above)
Comment (rossPatton) [reply _i18n/en.yml:27]: Yeah, I was worried about that myself. I just felt bad about removing it entirely.

**Pull request merged**